docs/receptor_mesh.md

```diff
@@ -0,0 +1,68 @@
+## Receptor Mesh
+
+AWX uses a Receptor mesh to transmit "user-space" unified jobs:
+ - jobs
+ - ad hoc commands
+ - inventory updates
+
+to the node where they run.
+
+https://github.com/ansible/receptor
```

Comment:
I think it would be best to remove this standalone link from here and convert the word `Receptor` in that very first sentence to `[Receptor](https://github.com/ansible/receptor)`